packages/debugger/lib/stacktrace/reducers.js

```diff
@@ -81,13 +81,14 @@ function lastPosition(state = null, action) {
   switch (action.type) {
     case actions.JUMP_IN:
     case actions.JUMP_OUT:
-    case actions.ETERNAL_CALL:
+    case actions.EXTERNAL_CALL:
```

Comment:
Btw, reminder for the future, this sort of thing should really be a separate PR.  But it's already in here so, whatever!

Reply:
I know, it was the one line that started all this 🤷🏻‍♂️